tests/unit/test_cli.py

```
@@ -223,6 +225,65 @@ def test_cli_context_window_override_environment_variable() -> None:
+        config = _unwrap_config(apply_cli_args(args))
+        assert config.anthropic_port == config.port + 1
+
+
+def test_apply_cli_args_updates_derived_anthropic_port_with_port_override() -> None:
+    """Port overrides should update the implied Anthropic compatibility port."""
+
+    def fake_load_config(
+        _config_path: Any | None = None,
+        *,
+        resolution: ParameterResolution | None = None,
+        environ: Any | None = None,
```

Ruff (0.13.3)

Unused function argument: `environ`

(ARG001)